executors/kv/kvexecutor.go

```diff
@@ -0,0 +1,136 @@
+package executor
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"sort"
+	"strings"
+	"sync"
+	"time"
+)
+
+// KVExecutor is a simple in-memory key-value store that implements the Executor interface for testing purposes.
+// It maintains an in-memory store and a mempool for transactions. I also add fields to track genesis initialization.
+
+type KVExecutor struct {
+	store   map[string]string
+	mu      sync.Mutex
+	mempool [][]byte
```

Comment:
how can we submit a tx by starting a node? seems to be missing the simple http server you had before. Could you also wire it up into run_node.go?